CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 * Add `setCountryCode` to `GooglePayRequest`
 * Add Google Pay support for Elo cards. 
+* Add new classes for 3DS2 UI Customization:
```

Comment:
Can we make a note of this in the migration guide as well?

Reply:
Added this to the migration guide 👍